source/dotnet/wholesale-api/CalculationResults/CalculationResults.Infrastructure/CalculationResultClient/CalculationResultClient.cs

```diff
@@ -68,7 +68,7 @@ public CalculationResultClient(HttpClient httpClient, IOptions<DatabricksOptions
         throw new NotImplementedException("GetAsync is not implemented yet");
     }
 
-    private async Task<DatabricksSqlResponse> SendSqlStatementAsync(string sqlStatement)
+    private async Task<Table> SendSqlStatementAsync(string sqlStatement)
```

Comment:
Perhaps (I don't find it natural that a send operation returns a result relating to executing the "thing" that was sent)
```suggestion
    private async Task<Table> ExecuteSqlStatementAsync(string sqlStatement)
```